JavaScript/JavaScriptSDK/TelemetryContext.ts

```diff
@@ -65,10 +67,22 @@ module Microsoft.ApplicationInsights {
         public session: Context.Session;
 
         /**
+        * Custom properties added to all telemetry items (implementation of the TelemetryInitializer concept).
+        */
+        public properties;
+
+        /**
+        * Custom measurements added to all telemetry items (implementation of the TelemetryInitializer concept).
+        */
+        public measurements;
+        
+        public onInitializeTelemetry: (envelope: Telemetry.Common.Envelope) => boolean;
+
```

Comment:
Hmm - what can user do to the envelope? Can they see what telemetry type this is? Can they change and add properties and measurements? Would it be better to have telemetry type string passed into onInitializeTelemetry as separate argument? Do we need to expose telemetry type constants for the user they can do this check?